src/pages/TxIndexPage.re

```diff
@@ -344,6 +344,27 @@ let make = (~txHash) => {
                    }}
                 </Col>
               </Row>
+              <Row alignItems=Row.Center>
+                <Col col=Col.Four mbSm=8>
+                  <Heading
+                    value="Fee Payer"
+                    size=Heading.H4
+                    weight=Heading.Thin
+                    color={theme.textSecondary}
+                  />
+                </Col>
+                <Col col=Col.Eight>
```

Comment:
Can we not show this section if there is no fee_payer?
When there is no fee_payer, it means the sender paid it themselves.